test/coordinator-test.js

```diff
@@ -0,0 +1,19 @@
+import { assert } from 'chai';
+import os from 'os';
+import sinon from 'sinon';
+import { getWorkerCount } from '../lib/coordinator';
+
+describe('coordinator', () => {
+  it('uses the correct number of cpus', () => {
+    const sandbox = sinon.sandbox.create();
+    sandbox.stub(os, 'cpus').returns([{}, {}, {}, {}, {}]);
```

Comment:
Hmm I generally try and stick to the 'no logic in tests' thing as much as possible - does asserting against the length of an object possibly violate this?